src/components/users/TicksModal.tsx

```diff
@@ -12,7 +12,7 @@ const TicksModal: React.FC<any> = ({ open, setOpen, setTicks, setOpenForm, climb
   }
   return (
     <Transition.Root show={open} as={Fragment}>
-      <Dialog as='div' className='relative z-10' initialFocus={cancelButtonRef} onClose={setOpen}>
```

Comment:
These z-values were needed so the breadcrumb nav bar didn't obscure the Tick modal.